doc/content/components/_index.md

```diff
@@ -2,4 +2,23 @@
 title: "Components"
 description: ""
 weight: 4
+menu:
+  main:
+    weight: 4
 ---
+
+TTN Stack is composed of components. The core components Network Server, Application Server and Join Server follow the LoRaWAN Network Reference Model. TTN Stack also contains an Identity Server, Gateway Server and Console.
```

Comment:
Find it redundant to say they are components in the stack, when the title says components.
```suggestion
The core components Network Server, Application Server and Join Server follow the LoRaWAN Network Reference Model. TTN Stack also contains an Identity Server, Gateway Server and Console.
```